lib/request.js

```diff
@@ -30,11 +30,6 @@ function buildRequest (R) {
 }
 
 Object.defineProperties(Request.prototype, {
-  req: {
-    get: function () {
-      return this.raw
-    }
-  },
```

Comment:
I would keep this getter with a deprecated warning.

Reply:
Issuing deprecation warnings in the same manner as Node core, such that `--no-deprecation` and --trace-deprecation` work, is the only way this PR will get a +1 from me. This will be a very big change in the API and hard dropping it makes us no better than another framework I will not name.